schemas/com.mparticle.snowplow/impression_event/jsonschema/1-0-0

```diff
@@ -0,0 +1,68 @@
+{
+	"$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+	"description": "Schema for an impression event",
+	"self": {
+		"vendor": "com.mparticle.snowplow",
+		"name": "impression_event",
+		"format": "jsonschema",
+		"version": "1-0-0"
+	},
+	"definitions": {
```

Comment:
We haven't used this approach before with schemas. Does this definitely work with Snowplow - have you tested it?

Reply:
I was going off what I saw in the SendGrid schema: https://github.com/snowplow/iglu-central/blob/master/schemas/com.sendgrid.sauna.commands/send_email/jsonschema/1-0-0#L16

Is that not correct?

Reply:
Oh nice, I hadn't seen that one! Cool, that should work then